src/plugins/auto/auto_schedule.hpp

```diff
@@ -18,6 +18,8 @@ struct AutoLoadContext {
     std::atomic<bool> isEnabled = {false};
     std::atomic<bool> isAlready = {false};
     std::atomic<bool> isLoadSuccess = {false};
+    std::atomic<bool> isReloadSuccess = {false};
+    std::atomic<bool> ableReload = {false};
```

Comment:
do we really need this flag？ looks like without this, a future/promise will be enabled, but this seems will not be harmful

Reply:
I see, it's about the future can only be used once. 
So why not just check the FALLBACKDEVICE enum, disable future/promise when the context is AutoLoadContextIndex::FALLBACKDEVICE

Reply:
Yes, this will be the latest modification :)